modules/adheseBidAdapter.js

```diff
@@ -20,11 +22,15 @@ export const spec = {
     if (validBidRequests.length === 0) {
       return null;
     }
+
     const { gdprConsent, refererInfo } = bidderRequest;
 
+    const adheseConfig = config.getConfig('adhese');
     const gdprParams = (gdprConsent && gdprConsent.consentString) ? { xt: [gdprConsent.consentString] } : {};
     const refererParams = (refererInfo && refererInfo.referer) ? { xf: [base64urlEncode(refererInfo.referer)] } : {};
-    const commonParams = { ...gdprParams, ...refererParams };
+    const globalCustomParams = (adheseConfig && adheseConfig.globalTargets) ? adheseConfig.globalTargets : {};
```

Comment:
we may want to clean the input using the `cleanTargets` funcsion as we do for the bid-level targets